eta-meeting-organizer-backend/src/main/java/hu/flowacademy/meetingorganizer/rest/MeetingRoomResource.java

```diff
@@ -0,0 +1,52 @@
+package hu.flowacademy.meetingorganizer.rest;
+
+import hu.flowacademy.meetingorganizer.persistance.model.MeetingRoom;
+import hu.flowacademy.meetingorganizer.service.MeetingRoomService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+@AllArgsConstructor
+public class MeetingRoomResource {
+
+    private MeetingRoomService meetingRoomService;
+
+    @GetMapping("/meetingrooms")
+    public ResponseEntity<List<MeetingRoom>> findAllMeetingRooms() {
+        List<MeetingRoom> meetingRooms = meetingRoomService.findAllMeetingRooms();
+        return new ResponseEntity<>(meetingRooms, HttpStatus.OK);
+    }
+
+    @GetMapping("/meetingrooms/{id}")
+    public ResponseEntity<MeetingRoom> findOne(@PathVariable Long id) {
+        MeetingRoom meetingRoom = meetingRoomService.findOneMeetingRoomById(id);
+        if(meetingRoom == null) {
+            return new ResponseEntity<>(HttpStatus.NOT_FOUND);
+        } else {
+            return new ResponseEntity<>(meetingRoom, HttpStatus.OK);
```

Comment:
Ha vannak olyan megfelelő HTTP response státuszt tartalmazó ResponseEntity-k, amiket lehet statikusan is kreálni, szerintem használjuk inkább azokat. I.e. `ResponseEntity.ok()`

Reply:
return meetingRoom == null ? ResponseEntity.notFound() : ResponseEntity.ok();